inst/ms/aggreCAT.qmd

```diff
@@ -169,16 +201,17 @@ datasets collected using such elicitation methods.
 
```

Comment:
I don't know exactly how Delphi operates so ignore this if I'm wrong, but aggreCAT _could_ apply to methods that use individual round one assessments, and then round two post-discussion is behavioural consensus. aggreCAT could then compare mathematical aggregations of round one with the behavioural consensus of round two.

Reply:
Yes that's a good point, I don't know if Delphi ever collects the 'original' unadulterated individual estimates, or whether it only collects the estimates post-behavioural consensus. I will look into it.

Reply:
From Bonnie:

> Hey Elliot - first, there's so much variation on how the Delphi technique is applied, that I wouldn't exclude using aggrecat for judgements elicited using some form of Delphi. To be honest, I'm not even sure that Delphi typically uses behavioural consensus, or if that's just a claim that we've / someone has made along the line and it has stuck! i actually just looked at a book chapter that I wrote for Bill Sutherland, and there we say that the final judgements are aggregated commonly using the mean or median :grimacing:. (this chapter was whipped up so quickly that it's very possible this wasn't really checked properly). In any case, yes i think it's safe to say that  judgements obtained with Delphi are indeed elicited over two or more rounds from each expert, and however they are combined at the end (behavioural or mathematical aggregation), they COULD be mathematically aggregated at any point, and I'm pretty sure they are.... i think some people call this the 'statistical group' judgement. Hope I haven't confused the matter further. I'm actually keen to look into the 'strict' Delphi approach a bit more myself, so I'll endeavour to look up the original paper tomorrow!